chat-commands.js

```diff
@@ -2450,60 +2450,63 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return;
-		if (toId(target) === 'show') return this.errorReply("You're looking for /showbl");
+		if (toId(target) === 'show') return this.errorReply(`You're looking for /showbl`);
 
 		target = this.splitTarget(target);
 		const targetUser = this.targetUser;
-		if (!targetUser) return this.errorReply("User '" + this.targetUsername + "' not found.");
+		if (!targetUser) {
+			return this.errorReply(`User ${this.targetUsername} not found.`) +
+			this.errorReply(`If you are trying to blacklist an offline account use /blacklistname.`);
```

Comment:
I'm surprised Travis isn't yelling at you fo-

_oh, the `+`_

wtf though

Reply:
Do you know what `+` means?

Reply:
Just put the `return` on the second line.

Reply:
uh yes, this is exactly how its done elsewhere on PS and is pretty standard

Reply:
Can you link to anywhere else in PS this is done?